### PR TITLE
Heck why did I do this online

### PR DESCRIPTION
**Pull request by danielgraycode**
Quickly changed the url for obvious reasons and also added RcCarRacer which I may or may not have forgotten to do a few days ago.

### CODE REVIEW

File: staff.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
                           <i class="fa fa-twitter"></i>
                       </span>
               </a>
-              <a class="nav-item" href="https://www.discord.gg/ssl">
+              <a class="nav-item" href="https://sslcommunity.io/sslcommunity">
```
Comment (Kelwing) [inline staff.html]: Is this link even valid? I don't see any change here that would cause that to redirect...

**Review by Kelwing — CHANGES_REQUESTED** [review]

File: staff.html (hunk 1 above)
Comment (danielgraycode) [reply staff.html]: Sorry mistake on my part, accidently wrote the wrong thing. Will fix now

**Review by danielgraycode — COMMENTED** [review]
Fixed the link

File: faq.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
                           <i class="fa fa-twitter"></i>
                       </span>
               </a>
-              <a class="nav-item" href="https://www.discord.gg/ssl">
+              <a class="nav-item" href="https://sslcommunity.io/sslcommunity">
```
Comment (Kelwing) [inline faq.html]: what about this one?

File: guidelines.html [hunk 3]
```diff
@@ -65,7 +65,7 @@
                           <i class="fa fa-twitter"></i>
                       </span>
               </a>
-              <a class="nav-item" href="https://www.discord.gg/ssl">
+              <a class="nav-item" href="https://sslcommunity.io/sslcommunity">
```
Comment (Kelwing) [inline guidelines.html]: this one?

File: index.html [hunk 4]
```diff
@@ -74,7 +74,7 @@
                           <i class="fa fa-twitter"></i>
                       </span>
               </a>
-              <a class="nav-item" href="https://www.discord.gg/ssl">
+              <a class="nav-item" href="https://sslcommunity.io/sslcommunity">
```
Comment (Kelwing) [inline index.html]: this guy too

**Review by Kelwing — CHANGES_REQUESTED** [review]

**Review by danielgraycode — COMMENTED** [review]
Ok done

**Review by Kelwing — APPROVED** [review]

**Pull request merged**